README.md

```diff
@@ -1,583 +1,53 @@
-# K8s Custom Resource and Operator For TensorFlow jobs
+# Kubernetes Operator For TensorFlow jobs
 
 [![Build Status](https://travis-ci.org/tensorflow/k8s.svg?branch=master)](https://travis-ci.org/tensorflow/k8s)
 [![Coverage Status](https://coveralls.io/repos/github/tensorflow/k8s/badge.svg?branch=master)](https://coveralls.io/github/tensorflow/k8s?branch=master)
 [![Go Report Card](https://goreportcard.com/badge/github.com/tensorflow/k8s)](https://goreportcard.com/report/github.com/tensorflow/k8s)
-[Prow Test Dashboard](https://k8s-testgrid.appspot.com/sig-big-data)
-[Prow Jobs](https://prow.k8s.io/?repo=tensorflow%2Fk8s)
+[![Prow Test Dashboard](https://img.shields.io/badge/prow-dashboard-blue.svg)](https://k8s-testgrid.appspot.com/sig-big-data)
+[![Prow Jobs](https://img.shields.io/badge/prow-status-blue.svg)](https://prow.k8s.io/?repo=tensorflow%2Fk8s)
 
 ## Overview
 
-TFJob provides a Kubernetes custom resource that makes it easy to
+The operator is a part of [Kubeflow](https://github.com/kubeflow/kubeflow)
```

Comment:
The new wording here makes it sound like Kubeflow is providing the custom resource

Reply:
🤔 Yeah, it does, I will update it.